spec/features/dashboard/index_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+describe 'As an authenticated user' do
+  describe "When I visit '/dashboard'" do
```

Comment:
Nice work with these tests - I think we should look into changing these hard-coded routes for path helpers before we turn it in